### ISSUE TITLE
File Exception error

### ISSUE THREAD

**Issue by SubhadeepJasu** [yumiris/SPV3.CLI#8]
System.IO.FileNotFoundException: Could not find file 'E:\Sync\SPV3_Private_Repo\0x00.bin'.

**Comment by SubhadeepJasu** [comment]
```
Unhandled Exception: System.IO.FileNotFoundException: Could not find file 'E:\Sync\SPV3_Private_Repo\0x00.bin'.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.File.InternalReadAllBytes(String path, Boolean checkHost)
   at System.IO.File.ReadAllBytes(String path)
   at SPV3.CLI.File.ReadAllBytes() in C:\TeamCity\buildAgent\work\4a8b3c07c2740361\src\SPV3.CLI\File.cs:line 82
   at SPV3.CLI.Manifest.Load() in C:\TeamCity\buildAgent\work\4a8b3c07c2740361\src\SPV3.CLI\Manifest.cs:line 69
   at SPV3.CLI.Kernel.VerifyMainAssets() in C:\TeamCity\buildAgent\work\4a8b3c07c2740361\src\SPV3.CLI\Kernel.cs:line 78
   at SPV3.CLI.Kernel.Bootstrap() in C:\TeamCity\buildAgent\work\4a8b3c07c2740361\src\SPV3.CLI\Kernel.cs:line 45
   at SPV3.CLI.Program.<>c.<Main>b__0_5() in C:\TeamCity\buildAgent\work\4a8b3c07c2740361\src\SPV3.CLI\Program.cs:line 44
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at SPV3.CLI.Program.<Main>g__Execute|0_0() in C:\TeamCity\buildAgent\work\4a8b3c07c2740361\src\SPV3.CLI\Program.cs:line 44
   at SPV3.CLI.Program.Main(String[] args) in C:\TeamCity\buildAgent\work\4a8b3c07c2740361\src\SPV3.CLI\Program.cs:line 74```

**Comment by MirisWisdom** [comment]
Resolved in 1671333766aa3aab450b4b8dc338040dfd87f31b:

https://github.com/yumiris/SPV3/blob/1671333766aa3aab450b4b8dc338040dfd87f31b/src/SPV3.CLI/Kernel.cs#L77-L86

Latest compiled binary in `bin.tar` should contain the fixes.
